convert_hf_to_gguf.py

```diff
@@ -3456,6 +3495,14 @@ def parse_args() -> argparse.Namespace:
         "--no-tensor-first-split", action="store_true",
         help="do not add tensors to the first split (disabled by default)"
     )
+    parser.add_argument(
+        "--metadata", type=Path,
+        help="Specify the path for an authorship metadata override file"
+    )
+    parser.add_argument(
+        "--get-outfile", action="store_true",
+        help="get calculated default outfile name"
```

Comment:
```suggestion
        help="print calculated output file name then exit"
```

I don't really understand the usage of this option, but IMO we can also have `--dry-run` option that just print out everything without writing to gguf file (print output file name, tensor names, etc)

Reply:
There's already a `--dry-run` option, which was introduced in #6942 to only print the split plan (also valid for a single-file output), which includes the output name, so I agree that `--get-outfile` is redundant.